src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -985,7 +964,7 @@ bool emberAfOperationalCredentialsClusterAttestationRequestCallback(app::Command
 
     attestationElementsLen = TLV::EstimateStructOverhead(certDeclSpan.size(), attestationNonce.size(), sizeof(uint64_t) * 8);
 
-    if (!attestationElements.Alloc(attestationElementsLen))
+    if (!attestationElements.Alloc(attestationElementsLen + Crypto::kAES_CCM128_Key_Length))
```

Comment:
```suggestion
    if (!attestationElements.Alloc(attestationElementsLen + attestationChallenge.size()))
```
is more clear, I would think....